### PR TITLE
OSDOCS-2423 RHEL8 control hosts not yet supported

### PR DESCRIPTION
**Pull request by codyhoag**
https://issues.redhat.com/browse/OSDOCS-2423

Preview: https://deploy-preview-36128--osdocs.netlify.app/openshift-enterprise/latest/machine_management/adding-rhel-compute.html#rhel-preparing-playbook-machine_adding-rhel-compute

### CODE REVIEW

**Comment by codyhoag** [comment]
@gpei can you verify? Thanks!

**Comment by gpei** [comment]
@codyhoag thanks for the update, it lgtm. BTW, could we also clarify the machine we used to run playbook should be a RHEL-7 machine, such as in Line#9

`you must prepare a machine to run the playbook from. `
to
`you must prepare a RHEL-7 machine to run the playbook from. `

**Comment by gpei** [comment]
lgtm, thanks for the update

File: modules/rhel-preparing-playbook-machine.adoc [hunk 1]
```diff
@@ -15,11 +15,11 @@ Before you can add compute machines that use Red Hat Enterprise Linux as the ope
 
 .Procedure
 
-. Ensure that the `kubeconfig` file for the cluster and the installation program that you used to install the cluster are on the machine. One way to accomplish this is to use the same machine that you used to install the cluster.
+. Ensure that the `kubeconfig` file for the cluster and the installation program that you used to install the cluster are on the {op-system-base} 7 machine. One way to accomplish this is to use the same machine that you used to install the cluster.
```
Comment (sagidlow) [inline modules/rhel-preparing-playbook-machine.adoc:18]: Saw another PR yesterday that had it as `kubeConfig`. Should this be `kubeConfig` or `kubeconfig`?

**Review by sagidlow — COMMENTED** [review]
One question, otherwise LGTM.

File: modules/rhel-preparing-playbook-machine.adoc (hunk 1 above)
Comment (codyhoag) [reply modules/rhel-preparing-playbook-machine.adoc:18]: I believe there is a `kubeConfig` property that can be configured (e.g., [here](https://docs.openshift.com/container-platform/4.8/rest_api/config_apis/authentication-config-openshift-io-v1.html)). But in terms of the actual file, it's `kubeconfig` AFAIK (like [here](https://docs.openshift.com/container-platform/4.8/installing/installing_bare_metal/installing-bare-metal-network-customizations.html#cli-logging-in-kubeadmin_installing-bare-metal-network-customizations)).

Comment (sagidlow) [reply modules/rhel-preparing-playbook-machine.adoc:18]: Thanks for explaining that :)

**Pull request merged**